extensions/amp-story/1.0/amp-story-reaction.js

```diff
@@ -178,17 +187,23 @@ export class AmpStoryReaction extends AMP.BaseElement {
   }
 
   /** @override */
-  buildCallback() {
+  buildCallback(concreteCSS = '') {
     this.options_ = this.parseOptions_();
     this.rootEl_ = this.buildComponent();
-    this.element.classList.add('i-amphtml-story-reaction');
+    this.rootEl_.classList.add('i-amphtml-story-reaction');
+    toArray(this.element.attributes).forEach((attr) => {
+      if (Object.values(ReactionFlags).includes(attr.name)) {
+        this.rootEl_.setAttribute(attr.name, attr.value);
+      }
+    });
```

Comment:
Need to get the flags here (theme, chip-style...) so that the styles specified in amp-story-reaction.css get applied properly. Otherwise there would be no way to bring those changes to the shadow root